docs/src/development/local/_index.md

```diff
@@ -44,27 +44,30 @@ You will also notice a new directory in your project, `.platform/local`, which i
 
 ## Building the site locally
 
-Run the `platform build` command to run through the same build process as would be run on Platform.sh.  That will produce a `_www` directory in your project root that is a symlink to the currently active build in the `.platform/local/builds` folder. It should be used as the document root for your local web server.
+Run the `platform build` command to run through the same build process as would be run on Platform.sh.
+This creates `_www` directory in your project root that's a symbolic link to the currently active build in the `.platform/local/builds` folder.
+Run your local server from the `www` directory.
 
 ```bash
-~/htdocs/my-project $ platform build
-Building application myapp (runtime type: php)
-  Beginning to build ~/htdocs/my-project/project.make.
-  drupal-7.38 downloaded.
-  drupal patched with install-redirect-on-empty-database-728702-36.patch.
-  Generated PATCHES.txt file for drupal
-  platform-7.x-1.3 downloaded.
+$ platform build
+
+Building application app (runtime type: golang:1.17)
 Running post-build hooks
-Symlinking files from the 'shared' directory to sites/default
+Creating symbolic links to mimic shared file mounts
+  Symlinking .cache to .platform/local/shared/cache
 
-Build complete for application myapp
-Web root: ~/htdocs/my-project/_www
-~/htdocs/my-project $
+Build complete for application app
+Web root: <PATH_TO_PROJECT>/_www
 ```
 
-Because the `platform build` command will run locally, any runtime or tools used in your build process need to be available in your local environment, or the build will fail.  It may also result in side effects, such as the installation on your local computer of packages referenced in your `dependencies` block.
+Because the `platform build` command runs locally,
+you need to have any runtime or tools used in your build process available in your local environment
+or the build fails.
 
-If that is undesirable, a local virtual machine will let you create an enclosed local development environment that won't affect your main system.
+The process may also result in side effects,
+such as the installation on your local computer of packages referenced in your `dependencies` block.
+If you don't want that, use a local virtual machine
+as an enclosed local development environment that doesn't affect your main system.
 
 ## Running the code
 
```

Comment:
Maybe we could add a note about encryption?
From:
> it can be quite slow as all communication with the database or cache server will need to travel from your computer to Platform.sh’s servers.
To:
> it can be quite slow as all encrypted communication with the database or cache server will need to travel from your computer to Platform.sh’s servers.

And maybe add a link to: https://docs.platform.sh/security/encryption.html#data-in-transit

Reply:
If you add `encrypted` there, it makes it seem like there is some communication that is encrypted and some that isn't. But I believe the tethering would force all communication through the SSH tunnel. I'd hope it's explained on the tethering page, or will be when it's reworked.